src/js/components/ToggleButtonGroup/__tests__/ToggleButtonGroup-test.tsx

```diff
@@ -0,0 +1,490 @@
+    render(
+      <Grommet>
+        <ToggleButtonGroup options={['one', 'two']} defaultValue="one" />
+      </Grommet>,
+    );
+
+    const activeToggleButton = screen.getByRole('radio', { name: /one/i });
+    // Assert that the active toggle button exists
+    expect(activeToggleButton).toBeInTheDocument();
+    // Assert that the aria-checked attribute of the active toggle button is true
+    expect(activeToggleButton).toHaveAttribute('aria-checked', 'true');
+  });
+
+  test('Should render defaultValue as active when options is array of objects', () => {
+    render(
+      <Grommet>
+        <ToggleButtonGroup
+          options={[
+            { label: 'Choice 1', value: 'c1' },
+            { label: 'Choice 2', value: 'c2' },
+            { label: 'Choice 3', value: 'c3' },
+          ]}
+          defaultValue="c2"
+        />
+      </Grommet>,
+    );
+    const activeToggleButton = screen.getByRole('radio', {
+      name: /Choice 2/i,
+    });
+    // Assert that the active toggle button exists
+    expect(activeToggleButton).toBeInTheDocument();
+    // Assert that the aria-checked attribute of the active toggle button is true
+    expect(activeToggleButton).toHaveAttribute('aria-checked', 'true');
+  });
+
+  test('Should put tab focus on first button when no value', async () => {
+    const user = userEvent.setup();
+
+    const { container } = render(
+      <Grommet>
+        <ToggleButtonGroup options={['one', 'two', 'three']} />
+      </Grommet>,
+    );
+
+    const focusToggleButton = screen.getByRole('radio', {
+      name: /one/i,
+    });
+
+    expect(focusToggleButton).not.toHaveFocus();
+    expect(document.body).toHaveFocus();
+
+    await user.tab();
+    expect(focusToggleButton).toHaveFocus();
+    expect(document.body).not.toHaveFocus();
+
+    expect(focusToggleButton).toHaveStyleRule(
+      'box-shadow',
+      '0 0 2px 2px #6FFFB0',
+      {
+        modifier: ':focus',
+      },
+    );
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  test('Should put tab focus on active button when value', async () => {
+    const user = userEvent.setup();
+
+    const { container } = render(
+      <Grommet>
+        <ToggleButtonGroup
+          options={['one', 'two', 'three']}
+          defaultValue="three"
+        />
+      </Grommet>,
+    );
+
+    const focusToggleButton = screen.getByRole('radio', {
+      name: /three/i,
+    });
+
+    expect(focusToggleButton).not.toHaveFocus();
+    expect(document.body).toHaveFocus();
+
+    await user.tab();
+    expect(focusToggleButton).toHaveFocus();
+    expect(document.body).not.toHaveFocus();
+
+    expect(focusToggleButton).toHaveStyleRule(
+      'box-shadow',
+      '0 0 2px 2px #6FFFB0',
+      {
+        modifier: ':focus',
+      },
+    );
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  test('Should allow caller to clear all values in controlled scenarios', () => {
```

Comment:
Now that we changed the default behavior, let's update this test to be `should allow caller to enforce a value in controlled scenarios`

Reply:
Can use the code from your "Controlled" storybook example.

Reply:
I updated this however since onChange type is wrong Im going to comment out.